index.js

```diff
@@ -141,6 +145,39 @@ function wrap (argv, env, workingDir) {
   return unwrap
 }
 
+// handle special cases of shebang, e.g., the
+// 2>/dev/null; exec "`dirname "$0"`/node" "$0" "$@"
+// approach used in Node 5.8.0.
```

Comment:
As mentiond in the nyc issue, this [shebang replacement](https://github.com/nodejs/node/blob/33c27f8fcffd7b0c6d609fdd4503b4f7a3c45bcd/tools/install.py#L107-L117) only applies to the `npm` executable – I think it would be a good idea to mention that here, otherwise the comment might be a little confusing?

Also, this “special” shebang approach itself is quite old, it’s the combination of it with the npm version shipped with v5.8.0 that creates the issue.